SMA_FluxExtraction.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>When building this 2D array, it could be useful to explain its function within the tutorial. This is so that the user can edit the internal parameters to fit their own use cases.</p><p>Karto explained that the 2D array allows us to easily mask out regions in the process of calculating the signal RMS, which is used in later steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>